src/components/Chart/Chart.tsx

```diff
@@ -203,11 +203,11 @@ const Chart = ({ digitalChannelsEnabled = false }) => {
 
     const digitalChannelsToCompute = useMemo(
         () =>
-            !zoomedOutTooFarForDigitalChannels && showDigitalChannels && bits
+            !zoomedOutTooFarForDigitalChannels && showDigitalChannels && hasBits
                 ? digitalChannelsToDisplay
                 : [],
         [
-            bits,
+            hasBits,
```

Comment:
After testing no we cannot remove the external variable as we need to force react to recompute the memo when hasBits changes and on the initial render this is false while later on it will be true and the other dependancies would not have changed :/ this is not ideal as it is and it should be move to so redux states. will be adding a comment